### PR TITLE
DOC: add readme for fov builder tool

### PR DESCRIPTION
**Pull request by ambrosejcarr**
Adds a README.md file for the FOV builder tool.

### CODE REVIEW

**Comment by ttung** [comment]
I think if you rebase, this will unbreak the tests.  Not that it matters since it's a doc.

File: docs/fov-builder.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+## Building Synthetic SpaceTx-specification compliant experiments
+
+starfish provides a tool to construct example datasets that can be used to test software for use with our formats.
```
Comment (ttung) [inline docs/fov-builder.md]: inconsistent use of line wrapping. would prefer soft wrapping.

File: docs/fov-builder.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  --nuclei-dimensions NUCLEI_DIMENSIONS
+                        Dimensions for the nuclei images. Should be a json
+                        dict, with r, c, and z as the possible keys. The value
+                        should be the shape along that dimension. If a key is
+                        not present, the value is assumed to be 0.
+```
+
+### Examples:
+
+Build a 3-field of view experiment with 2 channels and 8 hybridization rounds per primary image stack that samples z 30 times.
```
Comment (ttung) [inline docs/fov-builder.md]: inconsistent wrapping.

Comment (ambrosejcarr) [reply docs/fov-builder.md]: I hit enter at the end of a sentence. I am consistent. 

You would prefer that I... hit enter when I get to 100 characters?

Comment (ttung) [reply docs/fov-builder.md]: Interesting, I didn't know that markdown only started paragraphs after a blank line.

**Pull request merged**